components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/delete-confirm-dialog.js

```diff
@@ -171,6 +174,11 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone', 'file_browser',
                                                             } 
                                                         },
                                                         function (data) {
+                                                            var message = {
```

Comment:
we r not doing anything for message . Remove the log file and pass the message to outputcontroller so that it shows on editor console

Reply:
Please check for other places where we have not done it too